router/image/templates/nginx.conf

```diff
@@ -39,6 +39,10 @@ http {
     gzip_vary {{ or (getv "/deis/router/gzipVary") "on" }};
     {{ end }}
 
+    # DNS servers to query for address resolution. Needed to resolve OCSP responder.
+    resolver 8.8.8.8 8.8.4.4 valid=300s;
```

Comment:
Could parse `/etc/resolve.conf` and grab the [authoritative nameserver configuration](http://wiki.alpinelinux.org/wiki/Configure_Networking#Configuring_DNS) then dump that into an etcd key so it's accessible from the template?